core/targets.py

```
@@ -431,3 +431,412 @@ def _life_stage_warnings(age: int, life_stage: LifeStage, lang: str = "en") -> L
         warnings.append({"code": "child", "message": M["child"].get(lang, M["child"]["en"])})
 
     return warnings
+
+
+def calculate_bmr(
+    age: int,
+    weight: float,
+    height: int,
+    gender: str,
+    body_fat: float = None,
```

Ruff (0.13.3)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)